UPGRADING.md

```diff
@@ -12,27 +12,31 @@ Please make sure:
 - If you have enabled strict types that you are calling a function or a method with the correct type.
 - If you extend a class of Guzzle and you are overriding a method make sure you add the return type and type hints of the overriden method.
 
-Moreover:
-- All internal native functions calls of Guzzle are now prefixed with a slash. 
-This change makes it impossible for the users of the library to change the behavior of any of these functions
-
-Before:
-`curl_version()`
-
-After:
-`\curl_version()`
- 
+Other backwards compatibility breaking changes:
 - Class `GuzzleHttp\UriTemplate` is removed.
 - Class `GuzzleHttp\Exception\SeekException` is removed.
 - Classes `GuzzleHttp\Exception\BadResponseException`, `GuzzleHttp\Exception\ClientException`, `GuzzleHttp\Exception\ServerException` can no longer be initialized with an empty Response as argument.
 - Constant `GuzzleHttp\ClientInterface::VERSION` is removed.
 - Function `GuzzleHttp\Exception\RequestException::getResponseBodySummary` is removed. Use `\GuzzleHttp\Psr7\get_message_body_summary`
 as an alternative
 - Function `GuzzleHttp\Cookie\CookieJar::getCookieValue` is removed.
-- Request option `exception` which was an alias of `http_errors` is removed.
-- Request option `save_to` which was an alias of `sink` is removed.
+- Request option `exception` is removed. Please use `http_errors`.
+- Request option `save_to` is removed. Please use `sink`. 
+
+#### Native functions calls
+
+All internal native functions calls of Guzzle are now prefixed with a slash. 
+This change makes it impossible for the users of the library to change the behavior of any of these functions
+
+```php
+// Before:
+`curl_version()`;
+
+// After:
+`\curl_version()`;
+```
 
-For the full diff you can check [here](https://github.com/guzzle/guzzle/compare/6.5.0..7.0.0)
+For the full diff you can check [here](https://github.com/guzzle/guzzle/compare/6.5.0..master)
```

Comment:
This change should be reverted when we released 7.0.0. 

(Sorry @gmponos for suggesting to change this in your original PR)

Reply:
No problem.. should be changed to 6.5.2 also we need to add changes of 6.5.2 to master

Reply:
> also we need to add changes of 6.5.2 to master
https://github.com/guzzle/guzzle/pull/2509

Reply:
They are already included.